dask_geopandas/hilbert_distance.py

```diff
@@ -0,0 +1,261 @@
+import geopandas
+import numpy as np
+
+# from numba import jit # optional - do we want numba as a dependency for dask-geopandas?
+# ngjit = jit(nopython=True, nogil=True)
```

Comment:
As discussed on the meeting yesterday: let's simply use numba for now (and we can update later to make it imported lazily / make it an optional dependency). 

For that, you will also need to add numba to the dependencies in the different environment yaml files at https://github.com/geopandas/dask-geopandas/tree/master/continuous_integration/envs for the CI tests

Reply:
resolved in bc9b632